features/tests/test_api_mutations.py

```diff
@@ -0,0 +1,155 @@
+"""
+
+!- Draft status
+!- Draft is not available in the API
+!- Draft can be filtered in admin
+
+- Source type for things coming in from API
+- Unique source id, uuid?
+
+"""
+from copy import deepcopy
+
+import pytest
+
+from categories.tests.factories import CategoryFactory
+from features.enums import Visibility
+from features.models import Feature
+from features.tests.factories import TagFactory
+
+
+@pytest.fixture(autouse=True)
+def autouse_db(db):
+    pass
```

Comment:
might be useful to put it in `features/tests/conftest.py`. or do you only need it here?

Reply:
I was thinking about doing that, there's a few test modules that don't use it.. but majority of tests need it.. so maybe it makes more sense to have it in the `conftest.py` :+1: